### ISSUE TITLE
About the new ECCV paper

### ISSUE THREAD

**Issue by Owen-Liuyuxuan** [garrickbrazil/M3D-RPN#41]
Thanks for you great work and I have been following your works for a while.

I am a bit curious about some issues about your ECCV new paper.

It seems that video prediction requires ground-truth ego-motion of the camera, at least in the training set. But I am not sure whether KITTI provides such ground truth labels for the detection dataset?

Where could I find it? Sorry for such a naive question.

**Comment by brazgardian** [comment]
Hi Owen, we use the devkit within KITTI on the raw dataset to produce the ego-motion training ground truths (derived from the IMU/oxts). We will release this code in addition to the rest before ECCV on Aug 23!

**Comment by Owen-Liuyuxuan** [comment]
Thanks; Really looking forward to that!
